src/Storage/Commands.Storage/File/Cmdlet/SetAzureStorageFileContent.cs

```diff
@@ -163,14 +163,24 @@ public override void ExecuteCmdlet()
             }
             catch (StorageException e)
             {
-                if (e.RequestInformation != null &&
-                    e.RequestInformation.HttpStatusCode == (int)HttpStatusCode.BadRequest &&
-                    e.RequestInformation.ExtendedErrorInformation == null)
+                if (e.RequestInformation.HttpStatusCode == (int)HttpStatusCode.Forbidden)
```

Comment:
@blueww we should do the same null-check here for `e.RequestInformation`. Also, do we need to check the value of `e.RequestInformation.ExtendedErrorInformation`?

Reply:
Add null-check for e.RequestInformation.
Since we don't print e.RequestInformation.ExtendedErrorInformation here, don't need to check the value of it.